### ISSUE TITLE
Attachments for Another Layer

### ISSUE THREAD

**Issue by woswald** [agrc/daq-web-framework#44]
Would it be feasible to set up the "Permits (as of Nov. 1, 2016)" layer with the ability for attachments from edocs and other attachments in the same way that the "2014 O&G Facilities" layer is currently setup?

**Comment by steveoh** [comment]
![anything is possible](https://media.tenor.co/images/ef8b2905ea528e937992bcb995528077/raw)

**Comment by woswald** [comment]
LOL.

Let me rephrase. With where we are at in the project (time and money wise) how big of a deal would adding this be? Few hours worth of work, or blow all of our remaining budget?

**Comment by steveoh** [comment]
I can review the code and see how much effort it would take to modify that part.

#### A couple questions

Would the permits use the same queries from edocs or would they need to be unique for the layer?

**Comment by woswald** [comment]
Same queries using the AI#.

**Comment by steveoh** [comment]
I'll put this on @brigsz radar and we can chat about the budget, time, effort, and the existing issues #42 and #43

**Comment by woswald** [comment]
Sounds good. Thanks!

**Comment by steveoh** [comment]
Enable Edocs tab for permits layer

**Comment by steveoh** [comment]
To make this happen for the facilities we use the AI number to query edocs. Permit does not have that field but seems to be calling SITE_ID as the AI via an alias?  Does that seem correct @woswald and will that work with edocs?

**Comment by woswald** [comment]
Yes, the SITE_ID should be the same thing as AI#.  I assume as long as it can find those numbers in edocs it should be fine, and they should all be in there.  If they aren't they may just not have any documents.  Or if there is a problem with the field being named SITE_ID and just having an alias of AI#, let me know and I can try to republish that layer with the field renamed rather than using an alias.  Let me know if this answered your question.

**Comment by steveoh** [comment]
It would be quite a bit more straight forward if the layer had the same schema around AI number. I can make it work either way though.

**Comment by woswald** [comment]
Ok. I will republish that layer with that field renamed.  This won't interfere with anything else, will it?

**Comment by steveoh** [comment]
It might break the attributes

**Comment by woswald** [comment]
Still worth changing?

**Comment by steveoh** [comment]
Let me see how it goes without it

**Comment by woswald** [comment]
Ok, I'll just wait to hear back.

**Comment by steveoh** [comment]
@woswald, Will you please enable attachments/uploads for the permits layer.

**Comment by woswald** [comment]
It shows as being enabled on AGOL:
![image](https://cloud.githubusercontent.com/assets/7386943/26474831/b673d538-4170-11e7-9b54-18140cd8afb0.png)

**Comment by steveoh** [comment]
I'm getting `No permission to edit the specified feature.` when trying to add an attachment. Isn't the checkbox on the `settings` tab?

**Comment by woswald** [comment]
Attachments was turned on, but editing was off.  See if it works now.

**Comment by steveoh** [comment]
Much better. Thanks.

**Comment by woswald** [comment]
Great! NP.

**Comment by steveoh** [comment]
Do you know of any permits that have associated data in Edocs?
